oauth2_http/java/com/google/auth/oauth2/IamUtils.java

```diff
@@ -109,11 +112,11 @@ private static String getSignature(String serviceAccountEmail, Credentials crede
       Map<String, Object> error = OAuth2Utils.validateMap(responseError, "error", PARSE_ERROR_MESSAGE);
       String errorMessage = OAuth2Utils.validateString(error, "message", PARSE_ERROR_MESSAGE);
       throw new IOException(String.format("Error code %s trying to sign provided bytes: %s",
-              statusCode, errorMessage));
+                statusCode, errorMessage));
```

Comment:
continuation lines are indented 4 spaces, not 8. This is one of the areas where Google style deviates from Oracle style.

Reply:
ok, i think i got the indents here right; PTAL

Reply:
doesn't look right. Are you using tabs?